server/graphql/contact_form/src/mutations/insert.rs

```diff
@@ -0,0 +1,105 @@
+use async_graphql::*;
+use graphql_core::standard_graphql_error::StandardGraphqlError;
+use graphql_core::{standard_graphql_error::validate_auth, ContextExt};
+use repository::contact_form_row::ContactFormRow;
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    contact_form::{InsertContactForm as ServiceInput, InsertContactFormError as ServiceError},
+};
+
+#[derive(InputObject)]
+#[graphql(name = "InsertContactFormInput")]
+pub struct InsertContactFormInput {
```

Comment:
NIT: don't need the `#[graphql(..)]` renames now that the struct name is the same 😁

Reply:
Applied!